<launch>
  
  <include file="$(find smb_description)/launch/debug.launch">
    <arg name="visualize" value="1"/>
  </include>

  <include file="$(find map)/launch/continuous_tag_detection.launch"/>

  <node name="mapper" pkg="map" type="node.py" output="screen">
    <param name="pc_map_topic" value='/icp_node/icp_map'/>
    <param name="lidar_topic" value='/rslidar_points'/>
    <param name="tag_topic" value='/tag_detections'/>
    <param name="node_name" value='map'/>
    <param name="output_dir" value='/home/oilter/Courses/SemesterProject/catkin_ws/src/map'/>
  </node>
  
  <arg name="rosbag_path" default="/home/oilter/Courses/SemesterProject/RosBags/Mapping.bag"/>
  <node pkg="rosbag" type="play" name="player" output="screen" args="-d 5 -r 0.8 $(arg rosbag_path)" required = "true"/>

</launch>